### PR TITLE
feat: Add footer links for sourcecode and status page

### LINKED ISSUE #38: Add link to status page
as link rel="status" and in footer.

### LINKED ISSUE #54: Add link to GH repo to footer
And maybe some GitHub issues by-label

### PR DESCRIPTION
**Pull request by pke**
image

Closes #54, Closes #38

### CODE REVIEW

File: site/_layouts/default.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta name="description" content="{{site.description}}">
     <meta name="author" content="Marcus Ilgner">
-    <meta http-equiv="Content-Security-Policy" content="script-src 'self';img-src 'self';style-src 'unsafe-inline' 'self';">
+    <meta http-equiv="Content-Security-Policy" content="script-src 'self';img-src 'self' img.shields.io;style-src 'unsafe-inline' 'self';">
```
Comment (milgner) [inline site/_layouts/default.html:8]: Should be a header ideally... :thinking:

File: site/_layouts/default.html [hunk 2]
```diff
@@ -29,7 +29,13 @@ <h2>{% t page.header %}</h2>
       <footer class="row footer">
         <nav class="column">
           <a href="{{ "/imprint" | prepend: site.baseurl }}" rel="about">{% t titles.imprint %}</a>
-          <a href="/humans.txt">humans.txt</a>
+          <a href="/humans.txt" title="{% t titles.humans %}">humans.txt</a>
+          <a href="https://github.com/milgner/keedrop" title="{% t titles.source.title %}">
```
Comment (milgner) [inline site/_layouts/default.html:33]:
```suggestion
          <a href="https://github.com/keedrop/keedrop" title="{% t titles.source.title %}">
```

File: site/_layouts/default.html (hunk 1 above)
Comment (pke) [reply site/_layouts/default.html:8]: I was sure to have added the shields img host!?

Comment (milgner) [reply site/_layouts/default.html:8]: I'll think about it... :thinking:

**Pull request merged**